### ISSUE TITLE
eb create fails when using short syntax for .ebextension config file

### ISSUE THREAD

**Issue by Llamapaca** [aws/aws-elastic-beanstalk-cli#505]
### Description

`eb create` is failing with following error when using short syntax for config file
```
AttributeError: 'str' object has no attribute 'get'
```

### Steps to reproduce

command: `eb create`
example file:
```
option_settings:
  aws:elasticbeanstalk:environment:
    LoadBalancerType: network
```

### Observed result

exact command: `eb create --single --verbose --debug`
```
if setting.get('namespace') == 'aws:elasticbeanstalk:environment' and setting.get('option_name') == 'LoadBalancerType':
AttributeError: 'str' object has no attribute 'get'
````
### Expected result

Config file to be accepted

### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)

1. OS: Amazon linux 2
2. EBCLI version: EB CLI 3.20.10

**Comment by rahulrajaram** [comment]
Thanks for reporting this issue. We're working on a fix for it.

**Comment by jelofsson** [comment]
Is there any progress on this?

**Comment by rahulrajaram** [comment]
Fixed in 3.21. Apologies for the delay.
